site.js

```diff
@@ -136,6 +136,18 @@ $(document).ready(function (){
       {content: '0.9', start: '2017-06-01', end: '2018-06-01', py2:true},
       {content: '0.10', start: '2017-12-01', end: '2018-12-01'},
     ],
+    'Astropy':[
+      {content: '1.0 LTS', start: '2015-02-19', end: '2017-06-01', py2:true},
+      {content: '1.1', start: '2015-12-11', end: '2016-06-23', py2:true},
+      {content: '1.2', start: '2016-06-23', end: '2016-12-01', py2:true},
+      {content: '1.3', start: '2016-12-01', end: '2017-06-01', py2:true},
```

Comment:
1.3 "start" (and the "end" of 1.2) should probably be ``'2016-12-16'``, as that's what we currently have as the release date on https://github.com/astropy/astropy/wiki/Release-Calendar